<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalcWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>741</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="background">
     <item row="1" column="4" rowspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string> Operações</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>-1</x>
          <y>29</y>
          <width>181</width>
          <height>112</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonAdd">
           <property name="text">
            <string>Soma</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonMult">
           <property name="text">
            <string>Multipilicação</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonSub">
           <property name="text">
            <string>Subtração</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonDiv">
           <property name="text">
            <string>Divisão</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QDoubleSpinBox" name="input1"/>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="userRole">
       <item>
        <property name="text">
         <string>Usuário Comum</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrador</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="userLabel">
       <property name="text">
        <string>Logado como:  </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="exitButton">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Sair</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="execButton">
       <property name="text">
        <string>Executar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="factor2Label">
       <property name="text">
        <string>Parcela 2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QDoubleSpinBox" name="input2"/>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="resultEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLabel" name="titleLabel">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;C A L C U L A D O R A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="factor1Label">
       <property name="text">
        <string>Parcela 1:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>Resultado:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>textEditResult</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRelatorios">
    <property name="title">
     <string>Relatórios</string>
    </property>
    <addaction name="actionByUser"/>
    <addaction name="actionAllUsers"/>
   </widget>
   <addaction name="menuRelatorios"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionByUser">
   <property name="text">
    <string>Relatório por Usuário</string>
   </property>
  </action>
  <action name="actionAllUsers">
   <property name="text">
    <string>Relatório de Todos os Usuários</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>input1</sender>
   <signal>valueChanged(double)</signal>
   <receiver>resultEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>input2</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>resultEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>CalcWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
